package.json

```diff
@@ -273,7 +278,8 @@
     "env-var": "^4.1.0",
     "moment": "^2.20.1",
     "opn": "^5.2.0",
-    "simple-git": "^2.38.0"
+    "simple-git": "^2.38.0",
+    "yarn": "^1.22.10"
```

Comment:
Isn't yarn a devDependency?

Reply:
I am not sure. I changed to another pc and do the normal `npm install` and it didn't run. I manually installed `yarn` to make it work

Reply:
Packages listed in `dependencies` will be shipped with the extension.
I don't think that users of the extension need the yarn binary.